components/scream/src/physics/p3/p3_ice_cldliq_wet_growth_impl.hpp

```diff
@@ -43,7 +43,7 @@ ::ice_cldliq_wet_growth(
   Spack dum1{0.};
 
   if (any_if.any()) {
-    qsat0 = physics::qv_sat( zerodeg,pres,0 );
+    qsat0 = physics::qv_sat( zerodeg,pres, physics::Polysvp1 );
```

Comment:
`qv_sat` has 4 arguments. The first three are required while the last one is optional. In the function above, we should replace `physics::Polysvp1` with `false`. `physics::Polysvp1` or `physics::MurphyKoop` is the last (optional) argument, which this function call here does not need.

Reply:
Ah good catch!